group_vars/all

```diff
@@ -17,5 +17,8 @@ freesurfer_version: 7.4.1
 # https://github.com/MRtrix3/mrtrix3/releases
 mrtrix3_version: 3.0.5
 
+# https://github.com/MRtrix3/mrtrix3.git . This module compiles latest mrtrix commit in main
+mrtrix3_dev_version: "{{ (lookup('ansible.builtin.url', 'https://api.github.com/repos/MRtrix3/mrtrix3/commits/HEAD', wantlist=True)[0] | from_json).commit.committer.date | regex_replace('T.*', '') | regex_replace('-','') }}"
+
 # minc-toolkit-extras
-minc_toolkit_extras_version : "{{ (lookup('ansible.builtin.url', 'https://api.github.com/repos/CoBrALab/minc-toolkit-extras/commits/HEAD', wantlist=True)[0] | from_json).commit.committer.date | regex_replace('T.*', '') | regex_replace('-','') }}"
+minc_toolkit_extras_version : "{{ (lookup('ansible.builtin.url', 'https://api.github.com/repos/CoBrALab/minc-toolkit-extras/commits/HEAD', wantlist=True)[0] | from_json).commit.committer.date | regex_replace('T.*', '') | regex_replace('-','') }}"
```

Comment:
Why does it look like this changing? I don't see any changes.

Reply:
Mmmm not sure. I merged master in to this barnch to solve conflicts. The branch didn't have that line but the merge inlcudes it.

Reply:
Please see https://blog.verslu.is/git/git-rebase/ for the preferred workflow for this

Reply:
I see. Got it. Tried in a local branch and seems good.

Do you mind if I experiment deleting latest merge commit and do the rebase?

Reply:
yes of course, that's what I'd like you to try :+1:

Reply:
Done! Hopefully works haha